crates/noseyparker/data/default/builtin/rules/dropbox.yml

```diff
@@ -0,0 +1,18 @@
+rules:
+
+- name: Dropbox Access Token
+  id: np.dropbox.1
+
+  pattern: |
+    (?x)
+    \b
+    (sl\.[a-zA-Z0-9_-]{140})
+    \b
```

Comment:
The `\b` anchor is going to be problematic here, because the preceding character class includes `_` and `-`. An occurrence that ends with `-` will not match that anchor properly. Instead, you could match against end-of-input or a character other than the preceding character class.

You can see another example of where I've had to do this in the [JWT rule](https://github.com/praetorian-inc/noseyparker/blob/2405622d9ec2568ac3b9ebb3d2c8c4ea1fa3257e/crates/noseyparker/data/default/builtin/rules/jwt.yml#L19).

```suggestion
    (?: $ | [^a-zA-Z0-9_-] )
```